CHANGELOG.md

```diff
@@ -45,6 +45,7 @@ Please also have a look at our
 
 ### Fixed
 
+- Fix return type annotation of `Value::isSize()` (#892)
```

Comment:
Not sure if we're just changing a DocBlock comment this warrants a changelog entry.